libraries/botbuilder-core/botbuilder/core/teams/teams_info.py

```diff
@@ -401,3 +408,29 @@ async def _get_member(
         return TeamsChannelAccount().deserialize(
             dict(member.serialize(), **member.additional_properties)
         )
+
+    @staticmethod
+    async def send_meeting_notification_async(
+        turn_context: TurnContext,
+        notification: MeetingNotificationBase,
+        meeting_id: Optional[str] = None,
+    ) -> MeetingNotificationResponse:
+        if meeting_id is None:
+            meeting_id = turn_context.activity.id
+        else:
+            raise ValueError(
+                "This method is only valid within the scope of a MS Teams Meeting."
+            )
+
+        if not notification:
+            raise Exception(f"{notification} is required.")
+
+        teams_client = TeamsInfo.get_teams_connector_client(turn_context)
+
+        try:
```

Comment:
Please follow the existing structure and avoid using try-finally blocks unless necessary.